WebRoot/tiles/aboutBody.jsp

```diff
@@ -30,7 +30,6 @@ Source code is available on <a href="http://github.com/NESCent/plhdb">GitHub</a>
 <img src="/images/photos/Baboon_AlbertsAltmann_small.jpg" style="float:right; margin-left: 15px; margin-bottom:15px;" />
 <ul>
 <li>Concept, content, and scientific requirements: <a href="http://www.nescent.org/science/awards_summary.php?id=19">the Evolutionary Ecology of Primate Life Histories Working Group</a>, Susan C Alberts (Duke University, PI), Karen B Strier (University of Wisconsin-Madison, PI) 
-<li>Database and application programming: Hilmar Lapp (NESCent), Xianhua Liu (NESCent)
```

Comment:
This will remain true, right? Let's not delete attributions, even if affiliations are no longer current. They were at the time.